### ISSUE TITLE
1KG

### ISSUE THREAD

**Issue by luoshengtangxiademao** [AIRI-Institute/GENA_LM#11]

**Comment by minjaf** [comment]
Hi luoshengtangxiademao,

Indeed, in the final dataset version we used all SNVs from population (without AF filter). This 
1) saves computation time, and 
2) all data is from (apparently) healthy donors, so these are "normal" variants in human genomes, and 
3) common SNVs anyway have higher chance to appear in the corpus (they are common ==> we have higher chance to sample them).

We will update the text in our next preprint version, which we are going to release soon.

**Comment by luoshengtangxiademao** [comment]
Thank you. I see. But I still have a question, if all SNVs are used, the amount of data is huge, will it cause data imbalance when training with multi-species data? Too much human data (variation data from humans) makes multi-species data less effective?

**Comment by minjaf** [comment]
Yes, you are right, this why we explicitly set human:nonhuman data ratio to ~50%:

human T2T (w\ reverse complement) ~56B bp
sampled SNPs ~424B bp
non-human data, 591B bp

Total ~1.1T base pairs / ~166B tokens

As a result, not all possible SNPs are in the training dataset.

**Comment by luoshengtangxiademao** [comment]
I have one last question, the SNP and multi-species data distribution gap is relatively large, whether the two kinds of data are fully mixed in shuffle during training? Or is it possible to train the two data separately in stages?

**Comment by minjaf** [comment]
We have separate models trained either on human-only or on human+multispecies data. Please see [this table](https://github.com/AIRI-Institute/GENA_LM#pre-trained-models) for details. Although we have different setups, we did not try to _exclude_ human data from training.
